### PR TITLE
fix: retrieve the ECS cluster name from secret

### LINKED ISSUE #455: Fix: update GitHub workflow deployment to use new ECS service ARN format
# Summary
AWS has migrated to a [new ARN format for ECS resources](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-account-settings.html#ecs-resource-ids).

```sh
# Container instance
Old: arn:aws:ecs:region:aws_account_id:container-instance/container-instance-id
New: arn:aws:ecs:region:aws_account_id:container-instance/cluster-name/container-instance-id

# Amazon ECS service
Old: arn:aws:ecs:region:aws_account_id:service/service-name
New: arn:aws:ecs:region:aws_account_id:service/cluster-name/service-name

# Amazon ECS task
Old: arn:aws:ecs:region:aws_account_id:task/task-id
New: arn:aws:ecs:region:aws_account_id:task/cluster-name/task-id
```

As a result, a recent [`deploy-containers` run](https://github.com/cds-snc/covid-alert-server/actions/runs/942836361) failed looking for the old ARN format.

# Tasks
- [ ] Update workflow to use new ARN format
- [ ] Look for other instances of hardcoded ECS ARNs

### PR DESCRIPTION
**Pull request by patheard**
# Summary
Now that there are multiple clusters within the account, there isn't a reliable way to select the correct one without some hardcoding.

A future improvement could involve using `terraform output` to pull the cluster name from the environment's tfstate.

# Changes
* Retrieve target ECS cluster name from GitHub secrets.
* Remove monolith Docker image build and push (no longer used by the Heroku test app).

# Note
This change will also be applied to the `Deploy to production` workflow in the Terraform repo.

Closes #455

### CODE REVIEW

File: .github/workflows/deploy-containers.yml [hunk 1]
```diff
@@ -97,7 +91,7 @@ jobs:
         with:
           task-definition: ${{ steps.taskdef.outputs.task-definition }}
           service: ${{ matrix.SERVICE_NAME }}
-          cluster: ${{ steps.cluster.outputs.name }}
+          cluster: ${{ secrets.STAGING_ECS_CLUSTER_NAME }}
```
Comment (CalvinRodo) [inline .github/workflows/deploy-containers.yml:94]: Why pull this from secret it's public info:
https://github.com/cds-snc/covid-alert-server-staging-terraform/blob/83cf1e2d95ada5aafe12411dd66bef08cb24463d/server/aws/variables.auto.tfvars#L20

We can just hardcode and remove a chance for an error.

**Review by CalvinRodo — CHANGES_REQUESTED** [review]
See comment

File: .github/workflows/deploy-containers.yml (hunk 1 above)
Comment (patheard) [reply .github/workflows/deploy-containers.yml:94]: I'd put it as a secret to make it updatable without changing the workflow, but not a problem to just hard code it in the file.

**Review by CalvinRodo — APPROVED** [review]
:taco::cat:

**Pull request merged**